libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/SyncManager.java

```diff
@@ -354,6 +354,9 @@ private void updateSync(SyncState sync, SyncState.Status status, int progress, S
                         }
                         attributes.put("syncId", sync.getId());
                         attributes.put("syncTarget", sync.getTarget().getClass().getName());
+                        if (sync.getEndTime() > 0 && sync.getStartTime() > 0) {
+                            attributes.put("runTime", sync.getEndTime() - sync.getStartTime());
```

Comment:
On android, startTime / duration are long and represent milliseconds.
On iOS, they are integers and represent seconds.
We need to make them consistent and hold the best precision possible.